### PR TITLE
Allow user to skip wizard configuration steps

### LINKED ISSUE #1947: [Wizard] Add "skip" ability to every step.
Can either be done with a button + timeout, or a checklist to select the steps in the beginning

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #1947 

https://github.com/bluerobotics/BlueOS/assets/1215497/0171e819-1dac-4ee8-8e38-fed9d96b9492

### CODE REVIEW

File: core/frontend/src/components/wizard/ActionStepper.vue [hunk 1]
```diff
@@ -49,6 +59,9 @@ export default Vue.extend({
       return 'mdi-alert'
     },
     getStepColor(config: Configuration) {
+      if (config.skip) {
+        return 'black'
```
Comment (joaoantoniocardoso) [inline core/frontend/src/components/wizard/ActionStepper.vue:63]: This looks a bit odd... Can we add an 'X' or a '/' over the icon maybe?

**Review by joaoantoniocardoso — APPROVED** [review]
The black icon looks odd to me, but can be personal taste. 

Another thing is that the configuration step is happening on the "Done" step, but it's just another detail.

Despite that
image

**Comment by patrickelectric** [comment]
> Another thing is that the configuration step is happening on the "Done" step, but it's just another detail.

How so ? The logic behind it did not change, done is only applied once everything is accomplished.

File: core/frontend/src/components/wizard/ActionStepper.vue (hunk 1 above)
Comment (joaoantoniocardoso) [reply core/frontend/src/components/wizard/ActionStepper.vue:63]: maybe gray for a neutral feel?

Comment (patrickelectric) [reply core/frontend/src/components/wizard/ActionStepper.vue:63]: > maybe gray for a neutral feel?

I did try gray, but the contrast was terrible on white mode. black was the one that worked fine for white and dark mode.

**Comment by joaoantoniocardoso** [comment]
> > Another thing is that the configuration step is happening on the "Done" step, but it's just another detail.
> 
> How so ? The logic behind it did not change, done is only applied once everything is accomplished.

I see, maybe the name of the step should be "Apply" instead of "Done".

**Comment by patrickelectric** [comment]
> > > > Another thing is that the configuration step is happening on the "Done" step, but it's just another detail.
> > > 
> > > 
> > > How so ? The logic behind it did not change, done is only applied once everything is accomplished.
> > 
> > 
> > I see, maybe the name of the step should be "Apply" instead of "Done".
> 
> There is no Done step, where are you seeing it ?
> 
> There is "Done" state, and the "apply" operation.
> 
> But this is not related to this PR at all, this is the logic that is already on the code.
> 
> Since the feature only implies on the skip operation, we can discuss that later if valid.

I see what you are implying now, you are talking about the stepper called Done, will update to Apply.

**Comment by patrickelectric** [comment]
image

**Pull request merged**